webhook_receiver/settings/__init__.py

```diff
@@ -145,8 +145,16 @@
 CELERY_TASK_ALWAYS_EAGER = not bool(CELERY_BROKER_URL)
 
 DATABASES = {
-    'default': env.db('DJANGO_DATABASE_URL',
-                      default="sqlite://:memory:"),
```

Comment:
By the way it just occurred to me that this default is broken; it should be `sqlite:///:memory:` (one additional slash).

Reply:
Yes `env.db()` should return a proper config dict or raise a `ImproperlyConfigured` exception, but here it didn't. I couldn't understand why. Now that you mention the error in the default, I see why. If `DB_URL` is not set:

```
$ DB_URL="" python3
>>> from environ import Env
>>> env = Env()
>>> env.db('DB_URL', default="sqlite://:memory:")
/home/foad/.virtualenvs/webhook_receiver/lib/python3.9/site-packages/environ/environ.py:548: UserWarning: Engine not recognized from url: {'NAME': '', 'USER': '', 'PASSWORD': '', 'HOST': '', 'PORT': '', 'ENGINE': ''}
  warnings.warn("Engine not recognized from url: {}".format(config))
{}
```
This is what I was getting. An empty dict. And this would of course break when in production settings we want to access the config properties of this dict. 
I will remove the changes and just fix the sqlite path error. That will do the trick. Nice catch! :+1:

Reply:
Well actually just setting a default sqlite path doesn't work either. We need to specify the engine as well to get a proper dict:
```
$ DB_URL="" python3
>>> from environ import Env
>>> env = Env()
>>> env.db('DB_URL', default="sqlite:///:memory:")
/home/foad/.virtualenvs/webhook_receiver/lib/python3.9/site-packages/environ/environ.py:548: UserWarning: Engine not recognized from url: {'NAME': '', 'USER': '', 'PASSWORD': '', 'HOST': '', 'PORT': '', 'ENGINE': ''}
  warnings.warn("Engine not recognized from url: {}".format(config))
{}

>>>env.db('DB_URL', default="sqlite:///:memory:", engine='django.db.backends.sqlite')
{'NAME': '', 'USER': '', 'PASSWORD': '', 'HOST': '', 'PORT': '', 'ENGINE': 'django.db.backends.sqlite'}
```
:thinking:

Reply:
Huh?

Pulling up a temporary venv and just installing `django-environ` into it, here's what I get:

```
$ python
Python 3.8.10 (default, Nov 26 2021, 20:14:08) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from environ import Env
>>> env = Env()
>>> env.db('DB_URL', default="sqlite:///:memory:")
{'NAME': ':memory:', 'USER': '', 'PASSWORD': '', 'HOST': '', 'PORT': '', 'ENGINE': 'django.db.backends.sqlite3'}
```

So, exactly what's expected.

`django-environ` version issue, perhaps? My venv has 0.8.1.

Reply:
Ok I tried with a fresh venv as well (`django-environ` version 0.8.1). If `DB_URL` is not set at all, I get the same result as you. But if `DB_URL=""`, I get an empty dict.

Reply:
Well, yeah, but that *should* be that way. If I _don't_ set an environment variable, I expect a default to be used. If I set it to the empty string instead, I am telling the application, "use the empty string for this value", overriding the default.

Reply:
I'm actually surprised that django-environ considers "Engine not recognized from url" just a warning. It should be an `ImproperlyConfigured` error.

Reply:
That makes sense.

Reply:
So, maybe like this:

```python
default_db = env.db('DJANGO_DATABASE_URL',
                    default="sqlite://:memory:")
if not default_db:
    # <insert comment why this can be empty>
    raise ImproperlyConfigured("<Insert message here>")
DATABASES = {
    'default': default_db,
# [...]
```
What do you think?

Reply:
Yes that is better. I'll update the PR.